app/models/concerns/Auditable.rb

```diff
@@ -0,0 +1,10 @@
+# Concern Audiable for Models
+Module Auditable.rb
+ extend ActiveSupport :: Concern
```

Comment:
Similar deal, you can pop out the spaces here and have it read `extend ActiveSupport::Concern`.